downloads/openml_expdb.sql

```diff
@@ -744,6 +760,19 @@ CREATE TABLE `task_io_types` (
 
 -- --------------------------------------------------------
 
+--
+-- Table structure for table `task_study`
+--
+
+CREATE TABLE `task_study` (
```

Comment:
Just wondering: why different tables for run_study and task_study, and not just a general table with a type column?

Reply:
Good point.